assets/js/base/components/label/index.tsx

```diff
@@ -1,27 +1,29 @@
 /**
  * External dependencies
  */
-import PropTypes from 'prop-types';
-import { Fragment } from 'react';
+import { Fragment } from '@wordpress/element';
 import classNames from 'classnames';
+import type { HTMLAttributes } from 'react';
+
+interface LabelProps {
+	label?: string;
+	screenReaderLabel?: string;
+	wrapperElement?: string;
+	wrapperProps?: HTMLAttributes< HTMLElement >;
+}
```

Comment:
Contrary to what we discussed in the meetup, I think we should be explicitly defining interfaces and implementing. In an internal slack convo:

> ..the inline object there is actually a type alias I believe. It will behave very similarly but interfaces are preferred whenever possible.

See also [this reference](https://github.com/microsoft/TypeScript/wiki/Performance#preferring-interfaces-over-intersections).

The implication being that interfaces can have some performance benefits.

I think this only applies in some scenarios, but given this seems to be the convention being pointed to in A8C implementation of TS and is easier to recommend over thinking about whether to use or not - I think it makes sense to default to defining interfaces.